css/css-contain/contain-size-breaks-001.html

```diff
@@ -31,18 +30,36 @@
 <p>This test passes if there is an orange rectangle below. If the shape is not a rectangle, the test fails.
 
 <article>
-<div>A A A A A</div>
+<div>A B C D E</div>
 </article>
 
 <!--
-If the element is monolythic, it may:
-* overflow and be a 1x5 em rectangle
-* be sliced and be 2 contigious 1x2.5 em rectangles, appearing as a single
-  2x2.5 em rectangle.
+As the element is monolithic, it would be sliced in two contiguous 2.5em rectangles.
```

Comment:
The ASCII Art is good, but the comment isn't quite right. should be

> As the element is monolithic, it should either:
> * Overflow vertically, appearing as a signle 1x5 em rectangle
>  [... diagram...]
> * Be sliced, with one slice in each column, appearing as  a single 2x2.5 em rectangle
>  [... diagram...]

In other words, in the case where it overflows, it isn't being sliced.

Reply:
Ok thanks for the clarification, I've updated the comment.